contrib/gitian/DOCKRUN.md

```diff
@@ -74,12 +74,19 @@ You can set other options for that script by setting the OPT variable when runni
 e.g.
 
 ```bash
-OPT="-j 8" ./dockrun.sh v0.17.2.3
+# Run build processes with 8 threads
+OPT="-j 8" ./dockrun.sh v0.17.3.0
 ```
 
 You can also examine the build and install logs by running a shell in the container, e.g.
 
 ```bash
+# Tail running logs
+docker exec -it gitrun /bin/bash
+tail -F builder/var/*.log
```

Comment:
Do you really want *.log here? That will open all existing log files (including finished ones from previous builds).

Reply:
Should it instead be install.log and build.log (without the OS specified)?

I'm just trying to make it simple for people to see progress, I don't think it really matters that it will print a few lines of old logs.

Reply:
OK, fair enough.

Reply:
OK, fair enough.